docs/python/reference/dbos-class.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: DBOS Class
+---
+
+The DBOS class is a singleton&mdash;you may only instantiate it once in a program's lifetime.
```

Comment:
"initiate" is a bit vague. We can say "you may only call its constructor exactly once in a program's lifetyle"

Reply:
If I understand it correctly, you always have to call `DBOS()` even if you're explicitly using it?

Reply:
Fixed